src/executor/stack/memory.rs

```diff
@@ -368,6 +368,8 @@ impl<'config> MemoryStackSubstate<'config> {
 	}
 
 	pub fn set_storage(&mut self, address: H160, key: H256, value: H256) {
+		#[cfg(feature = "print-debug")]
+		println!("    [SSTORE {address:?}] {key:?}:{value:?}");
```

Comment:
```suggestion
		println!("[SSTORE {address:?}] {key:?}:{value:?}");
```

Reply:
@aleksuss For me it's more noticeable for bid debug output with spaces.